### ISSUE TITLE
Renaming brews in user page

### ISSUE THREAD

**Issue by 5e-Cleric** [naturalcrit/homebrewery#3046]
### Your idea:

I think it would make sense to allow for renaming in your user page, and it is a step ahead for a better UX

**Comment by G-Ambatte** [comment]
TLDR: I agree on the individual level, but I think we need to overhaul the User (or List) Page functionality for Brew metadata manipulation.

---

Currently, there are four actions available to authors on each item in their User Page:

* Share
* Edit
* Download
* Delete

I agree that `Rename` makes sense as a new action, but I would also suggest that so does `Add Tag` and `Remove Tag`, and when Folders exist, so to would `Add To Folder` and `Remove From Folder`.
While `Rename` makes sense as an individual item function, the Tag and Folder editing functions also make sense as a multiple item function, which then implies some method of selecting multiple items simultaneously.

**Comment by 5e-Cleric** [comment]
Also, when folders are here, all this functions will need to be working on folder pages as well.

**Comment by ericscheid** [comment]
> I agree that Rename makes sense as a new action, but I would also suggest that so does Add Tag and Remove Tag, and when Folders exist, so to would Add To Folder and Remove From Folder.

The 5th action could be the Brew Properties icon, wherein many things could be done.

**Comment by dbolack-ab** [comment]
> The 5th action could be the Brew Properties icon, wherein many things could be done.

Essentially open the metadata panel?

**Comment by 5e-Cleric** [comment]
> > The 5th action could be the Brew Properties icon, wherein many things could be done.
> 
> Essentially open the metadata panel?

Yes, or a reduced version of one.

**Comment by 5e-Cleric** [comment]
After carefull consideration, its unnecessary and would need an effort for no reason we can use in other features like folders. Closing as not planned.
